Fix build with libxml2 2.12+:

error: implicit declaration of function 'xmlReadFile' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://gitlab.freedesktop.org/xdg/shared-mime-info/-/issues/219
https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/c918fe77e255150938e83a6aec259f153d303573
--- src/test-subclassing.c.orig
+++ src/test-subclassing.c
@@ -1,4 +1,5 @@
 #include <libxml/tree.h>
+#include <libxml/parser.h>
 #include <stdio.h>
 #include <string.h>
 
